src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1679,46 +1676,61 @@ describe('useCourseListPrice', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     useCourseRedemptionEligibility.mockReturnValue({ data: { listPrice: mockListPrice } });
-    useCourseMetadata.mockReturnValue(mockListPrice || getCoursePrice(baseCourseMetadataValue));
+    useCourseMetadata.mockReturnValue({ data: mockListPrice || getCoursePrice(baseCourseMetadataValue) });
```

Comment:
[clarification] It looks like the mock is now returning `data` to mimic the `useQuery` result, but it essentially saying the `useCourseMetadata` is only returning list price (e.g., `{ data: 100 }`).

Perhaps we try to access the args passed to the mocked `useCourseMetadata` (i.e., the `select` fn) and call that function in the test, vs. testing the `useCourseMetadata` hook in itself (where we're essentially asserting the input we provide equals the same expected output (vs testing the underlying logic).

Reply:
I went ahead and updated all the test related to useCourseListPrice 👍🏽